docs/1.getting-started/6.data-fetching.md

```diff
@@ -272,13 +272,13 @@ Using `<script setup lang="ts">` is the recommended way of declaring Vue compone
 
 ## Using `$fetch` directly
 
-There are instances where you may need to directly call the API. Nuxt 3 provides a globally available `$fetch` method using [unjs/ofetch](https://github.com/unjs/ofetch) (in addition to `fetch`) with the same API as the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).
+There are instances where you may need to call the API directly. Nuxt 3 provides a globally available `$fetch` method using [unjs/ofetch](https://github.com/unjs/ofetch) (in addition to `fetch`) with the same API as the [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).
 
-Using `$fetch` has a number of benefits, including:
+Using `$fetch` has several benefits, including:
```

Comment:
_(...) including:_ makes me expect a list, not necessarily standalone paragraphs. What about merging first two into one and leaving the last sentence as it is?

```suggestion
Using `$fetch` has several benefits. It will handle 'smartly' making direct API calls if it runs on the server or making a client-side call to your API if it runs on the client. (It can also handle calling third-party APIs.)

Plus, it comes with a set of convenience features, including automatically parsing responses and stringifying data.
```